extensions/amp-share-tracking/0.1/amp-share-tracking.js

```diff
@@ -163,4 +163,10 @@ export class AmpShareTracking extends AMP.BaseElement {
   }
 }
 
-AMP.registerElement('amp-share-tracking', AmpShareTracking);
+
+// Install the extension.
+// DO NOT SUBMIT: First, we must implement compiler rewrite path to unwrap
+// this code.
+AMP.extension('amp-share-tracking', () => {
```

Comment:
So, i looked around. And I think we definitely canNOT roll the whole module inside `AMP.extension`. Besides being an anti-pattern in our world, it'd be also very hard to do this given the multi-module code in many extensions.